pkg/operator/controller/gatewayclass/istio.go

```diff
@@ -0,0 +1,209 @@
+package gatewayclass
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	gatewayapiv1 "sigs.k8s.io/gateway-api/apis/v1"
+
+	sailv1 "github.com/istio-ecosystem/sail-operator/api/v1"
```

Comment:
Sailv1?

Reply:
Yeah, it's the v1 version of the Istio API from the sailoperator.io API group, so it makes sense to import it as sailv1.  I think it would be confusing to import it as something else.  The downstream version of the operator modifies the controller logic, but it's the same API (by and large).

Reply:
My question here and elsewhere is why are we using github.com/istio-ecosystem/sail-operator/api/v1 (upstream repo) rather than github.com/openshift-service-mesh/sail-operator/api/v1 (downstream).